primitives/src/lib.rs

```diff
@@ -22,6 +28,24 @@ pub type BucketId = u64;
 pub type ClusterNodesCount = u16;
 pub type StorageNodePubKey = AccountId32;
 pub type ActivityHash = [u8; 32];
+
+pub struct MergeActivityHash;
```

Comment:
this is not the place for it. primitives is only for traits and structs. define interface here and imlement it in pallet or in the runtime